### ISSUE TITLE
ROLES: Print , save, export dataset

### ISSUE THREAD

**Issue by TonyNolan** [gjwgit/rattle#727]
## Description

It would be nice to have an icon on the top right hand side of the dataset tab, which allowed to save, export, and print the dataset.

## Why

So that changes mad to the dataset can be exported to a local CSV file.

## Closing Criteria

- [ ] Next to the VIEWER button in the ROLES page add icon for SAVE AS, EXPORT and PRINT.
- [x] SAVE AS when tapped will save the current ds to a *CSV* filename that is prompted for
- [ ] EXPORT when tapped will (for @TonyNolan to suggest otherwise leave this out for now)
- [ ] PRINT when tapped will generate a PDF containing the full dataset as a table then prompt to print/save to file

**Comment by zheyxu** [comment]
For **Save** button,

<img width="1247" alt="Image" src="https://github.com/user-attachments/assets/5e4de266-ee89-460b-a219-ba3f298fc5a2" />

**Comment by gjwgit** [comment]
SAVE might be good enough for now. Thanks.

**Comment by gjwgit** [comment]
Once SAVE is merged, we will move EXPORT/PRINT to backlog.

**Comment by zheyxu** [comment]
Made pr https://github.com/gjwgit/rattleng/pull/986 for save function.
